Configuration.props

```diff
@@ -19,7 +19,8 @@
     <AndroidMxeInstallPrefix Condition=" '$(AndroidMxeInstallPrefix)' == '' ">$(AndroidToolchainDirectory)\mxe</AndroidMxeInstallPrefix>
     <AndroidSdkDirectory>$(AndroidToolchainDirectory)\sdk</AndroidSdkDirectory>
     <AndroidNdkDirectory>$(AndroidToolchainDirectory)\ndk</AndroidNdkDirectory>
-    <AndroidSupportedAbis Condition=" '$(AndroidSupportedAbis)' == '' ">host-$(HostOS):armeabi-v7a</AndroidSupportedAbis>
+    <AndroidSupportedDeviceAbis>armeabi-v7a</AndroidSupportedDeviceAbis>
```

Comment:
We haven't had this property before, right?

We have $(AndroidSupportedAbis) in Configuration.Override.props and that should be used instead. Or if that is going to be conflict with something, then Configuration.Override.props.in needs to be modified to rename this relevant property.